packages/components/pie-modal/src/index.ts

```diff
@@ -209,36 +208,55 @@ export class PieModal extends RtlMixin(LitElement) implements ModalProps {
         `;
     }
 
-    render () {
+    public render () {
```

Comment:
I'm unsure about `public`. I've seen examples from Lit using `protected` too - Doesn't need to block this PR but would be good to understand the 'recommended' access modifier here

Reply:
That's a good point. `public` is the default so it's probably helpful to have it there as a reminder, in case we do want to change it in future